propolis/src/inventory.rs

```diff
@@ -177,19 +172,35 @@ struct InventoryInner {
 }
 
 impl InventoryInner {
-    /// Registers a new entity with the inventory.
+    /// Registers a new entity (and its children) with the inventory.
     ///
     /// Returns an error if the object has already been registered.
     /// Returns an error if the parent is not registered.
```

Comment:
Is this comment still accurate since we don't take a `parent_id` parameter anymore?

Reply:
Will fix